config/README.md

```diff
@@ -1,102 +1,103 @@
+own.
 
-List existing images:
+TODO(): The linux foundation should provide a default docker registry, with
+golden build versions.
 
-    gcloud --project $CLOUD_PROJECT container images list
-
-List the tags on an existing image:
-
-    gcloud --project $CLOUD_PROJECT container images list-tags gcr.io/$CLOUD_PROJECT/http-gateway
-
-Build a new image:
-
-    docker build -f cmds/http-gateway/Dockerfile  . -t gcr.io/$CLOUD_PROJECT/http-gateway:$VERSION
-    docker build -f cmds/grpc-backend/Dockerfile  . -t gcr.io/$CLOUD_PROJECT/grpc-backend:$VERSION
-
-Push your new image to Google Container Registry:
-
-    docker push gcr.io/$CLOUD_PROJECT/http-gateway:$VERSION
-    docker push gcr.io/$CLOUD_PROJECT/grpc-backend:$VERSION
+Set the environment variable `DOCKER_URL` to your docker registry url endpoint.
 
 Use the `build.sh` script in this directory to build and push an image tagged
 with the current date and git commit hash.
 
 
 ## Creating a new Kubernetes cluster
 
-You need to create a Kubernetes cluster to run the cockroachdb instance and the
-gRPC and HTTP servers.  You can do this on any supported
-[cloud provider](https://kubernetes.io/docs/concepts/cluster-administration/cloud-providers/)
-or even on your own infrastructure.  Instructions for GCP are given below,
-consult the Kubernetes documentation for other providers.
+This secion discusses deploying a kubernetes service, although you can deploy 
+a DSS node however you like as long as it meets the above requirements. You can
+do this on any supported [cloud provider](https://kubernetes.io/docs/concepts/cluster-administration/cloud-providers/)
+or even on your own infrastructure.  Consult the Kubernetes documentation for
+which ever provider you choose.
 
-### Google Container Engine
 
-Create a new cluster in the given zone:
+### Deploying with Kubernetes
 
-    gcloud --project $CLOUD_PROJECT container clusters create $CLUSTER_NAME --zone $ZONE
-
-Fetch credentials for the cluster.  This populates your \~/.kube/config file
-and makes all future kubecfg commands target this cluster.
-
-    gcloud --project $CLOUD_PROJECT container clusters get-credentials $CLUSTER_NAME --zone $ZONE
-
-
-## Creating a new cockroachdb cluster
-
-1.  Create 5 static IP addresses.  How you do this depends on your cloud
-    provider.  3 IPs will be used for the 3 individual cockroachdb nodes, 1 IP
-    will be used to load balance amongst all the cockroachdb nodes, and 1 IP
-    will be used for the HTTPS frontend.  The HTTPS frontend IP will be used on
-    a Kubernetes Ingress, and if you're using Google Cloud it needs to be
-    created as a "Global" IP address.
+1.  Create your static IP addresses: 1 for each of your CockroachDB nodes (min
+    of 3) and 1 for the HTTPS Gateway's Ingress.  How you do this depends on
+    your cloud provider. Note: if you're using Google Cloud the HTTPS gateway
+    Ingress needs to be created as a "Global" IP address.
```

Comment:
ingress (capitalization)

Reply:
https://kubernetes.io/docs/concepts/services-networking/ingress/ capitalized in kube docs